pricing.js

```diff
@@ -55,22 +57,43 @@ function calculateLTDPrice(product, employee, selectedOptions) {
   return price
 }
 
+function calculateCommuterPrice(product, selectedOptions) {
+  const selectionCost = product.cost.find((cost) => {
+    return cost.type === selectedOptions.benefit
+  })
+  return selectionCost.price
+}
+
 function calculateProductPrice(product, employee, selectedOptions) {
   let price
   let employerContribution
 
   switch (product.type) {
     case 'volLife':
-      price = calculateVolLifePrice(product, selectedOptions)
-      employerContribution = getEmployerContribution(product.employerContribution, price)
-      return formatPrice(price - employerContribution)
+      price = this.calculateVolLifePrice(product, selectedOptions)
+      employerContribution = this.getEmployerContribution(product.employerContribution, price)
+      return this.formatPrice(price - employerContribution)
     case 'ltd':
-      price = calculateLTDPrice(product, employee, selectedOptions)
-      employerContribution = getEmployerContribution(product.employerContribution, price)
-      return formatPrice(price - employerContribution)
+      price = this.calculateLTDPrice(product, employee, selectedOptions)
+      employerContribution = this.getEmployerContribution(product.employerContribution, price)
+      return this.formatPrice(price - employerContribution)
+    case 'commuter':
+      price = this.calculateCommuterPrice(product, selectedOptions)
+      selectionCost = this.product.costs.find((cost) => {
+        return this.cost.type === selectedOptions.benefit
+      })
+      return selectedOptions.price
```

Comment:
You've got a good `calculateCommuterPrice` function that you call but then you never do anything with the price it returns. You then reimplement the `calculateCommuterPrice` which you already called and then you return `selectedOptions.price` which isn't a thing, `selectedOptions` is what was passed in with the user's selection. After you call `calculateCommuterPrice` you should be getting the employer contribution and using both to return a formatted final price just like the other two product types.